config.ru

```diff
@@ -1,5 +1,12 @@
 # This file is used by Rack-based servers to start the application.
 
+require 'rack'
+require './lib/foreman/telemetry_rack'
+require 'prometheus/middleware/exporter'
+
+use Foreman::TelemetryRack
+use Prometheus::Middleware::Exporter
```

Comment:
This should be used only when prometheus provider is being used.

Reply:
Chicken and egg, Telemetry hasn't been initialized at this moment, I don't think I should initialize is this early. Having that enabled means that Prometheus will respond at `/metrics` exposing no measures (basically an empty request).

Possible solution - move telemetry into separate bundler group and only if the gem is installed then we would use it here.